shim.js

```diff
@@ -5535,6 +5535,10 @@ require.define("/shim.coffee", function (require, module, exports, __dirname, __
         if (cb == null) cb = function() {};
         return cb(page.injectJs(js));
       },
+      clearMemoryCache: function(cb) {
```

Comment:
Where is the coffee script changes? You should make changes to cs and do a build.